cutlery/_wordpiece.pxd

```diff
@@ -1,7 +1,10 @@
 from cython cimport size_t
 from libcpp.string cimport string
 from libcpp.unordered_map cimport unordered_map
+from libcpp.vector cimport vector
 
 cdef class WordPieceProcessor:
-    cdef unordered_map[string, size_t] initial_pieces
-    cdef unordered_map[string, size_t] continuation_pieces
+    cdef unordered_map[string, size_t] initial_piece_to_id
+    cdef unordered_map[size_t, string] id_to_initial_piece
+    cdef unordered_map[string, size_t] continuation_piece_to_id
+    cdef unordered_map[size_t, string] id_to_continuation_piece
```

Comment:
For the id -> piece lookup, I think it's nicer to define a small struct along the lines of:

```
struct Piece {
    std::string piece;
    bool is_initial;
};
```

And then have a single mapping

```
    cdef vector[Piece] id_to_piece
```

Reply:
Done.